### PR TITLE
Changed question naming, Added API, Minor changes to homepage structure

### PR DESCRIPTION
**Pull request by DanRie19**

### CODE REVIEW

**Comment by DanRie19** [comment]
closes #57

File: client/src/components/Header/Header.jsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ const Header = () => {
           <div className="header__right">
             <div>
               <Button
+                id="button1"
```
Comment (ErnieAtLYD) [inline client/src/components/Header/Header.jsx:43]: This is the only place in your commit where you reference a `button1` id, so I'm assuming you have this referenced to a different part of the code sitting around somewhere?

**Review by ErnieAtLYD — APPROVED** [review]
Looks good to me, see comment :+1:

File: client/src/components/Header/Header.jsx (hunk 1 above)
Comment (DanRie19) [reply client/src/components/Header/Header.jsx:43]: I'll scrap it, was editing a button and decided to leave that for later,

**Pull request merged**